<launch>
  <arg name="gui" default="true" />

  <node name="infratec_driver" pkg="infratec_ros_driver" type="qt_infratec.py" respawn="true" />

  <node name="infratec_image_convert_mono" pkg="infratec_image_processing" type="irb_image_convert_mono.py" output="screen" >
    <param name="thermal_image" type="str" value="/infratec/image_raw"/>
  </node>

  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/infratec/image_converted_mono"/>
    <param name="autosize" value="true" />
  </node>

  <node name="image_view2" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/infratec/image_raw"/>
    <param name="autosize" value="true" />
  </node>
  
</launch>
